app-admin/passwordsafe/passwordsafe-1.09.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake-utils desktop flag-o-matic wxwidgets
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="libressl minimal qr test +xml xvkbd yubikey"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	net-misc/curl
+	sys-apps/util-linux
+	x11-libs/libXt
+	x11-libs/libXtst
+	x11-libs/wxGTK:${WX_GTK_VER}[X]
+	!libressl? ( dev-libs/openssl:0= )
+	libressl? ( dev-libs/libressl:0= )
+	!minimal? ( !!app-misc/pwsafe )
+	qr? ( media-gfx/qrencode )
+	xml? ( dev-libs/xerces-c )
+	yubikey? ( sys-auth/ykpers )"
+RDEPEND="${COMMON_DEPEND}
+	xvkbd? ( x11-misc/xvkbd )"
+DEPEND="${COMMON_DEPEND}
+	app-arch/zip
+	sys-devel/gettext
+	test? ( dev-cpp/gtest )"
```

Comment:
Note that with EAPI-7 these are all most likely BDEPEND, which also allows you to remove `COMMON_DEPEND` and just inherit `DEPEND` in `RDEPEND`.